### PR TITLE
Grant riemann the ability to snapshot and stop VMs

### PR DESCRIPTION
**Pull request by cnelson**
This is for https://github.com/18F/cg-product/issues/476

### CODE REVIEW

File: terraform/modules/iam_role/riemann_monitoring/role.tf [hunk 1]
```diff
@@ -0,0 +1,44 @@
+module "riemann_monitoring" {
+  source = ".."
+
+  role_name = "${var.role_name}"
+  iam_policy = <<EOF
+{
```
Comment (jmcarp) [inline terraform/modules/iam_role/riemann_monitoring/role.tf]: WDYT about moving this into a separate template file while we're at it? I'm tired of committing policies with invalid JSON 😄

**Comment by cnelson** [comment]
Moved the policy to a separate file

File: terraform/modules/iam_role/riemann_monitoring/policy.json [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+  "Version": "2012-10-17",
+  "Statement": [
+    {
+      "Effect": "Allow",
+      "Action": [
+        "logs:DescribeLogGroups",
+        "logs:DescribeLogStreams"
+      ],
+      "Resource": [
+        "arn:${var.aws_partition}:logs:${var.aws_default_region}:${var.account_id}:*"
```
Comment (jmcarp) [inline terraform/modules/iam_role/riemann_monitoring/policy.json]: For some reason, we don't need the `var.` prefix in templates: https://www.terraform.io/docs/providers/template/d/file.html

Comment (cnelson) [inline terraform/modules/iam_role/riemann_monitoring/policy.json]: prefixes removed

**Pull request merged**